README.md

```
@@ -1,104 +1,753 @@
+[💬 Discussions](../../discussions) • [🐛 Issues](../../issues) • [📢 Changelog](./CHANGELOG.md)
+
+</div>
+
+![separator](https://raw.githubusercontent.com/andreasbm/readme/master/assets/lines/rainbow.png)
+
+## 📄 License
+This project is licensed under the **MIT License** - see the [LICENSE](./LICENSE) file for full details.
 
-dbLogger.debug('Connecting to database...');
-dbLogger.info('Query executed successfully');
 ```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)